conbench/templates/run.html

```diff
@@ -7,9 +7,22 @@
       {% if run_id %}<li class="breadcrumb-item active" aria-current="page">{{ run_id }}</li>{% endif %}
     </ol>
   </nav>
-  {% if compare_runs_url %}
+  {% if compare_to_baseline_urls['parent'] %}
     <p class="fs-5">
-      <i class="bi bi-file-diff"></i> <a href="{{ compare_runs_url }}">compare to baseline</a>
+      <i class="bi bi-file-diff"></i> <a href="{{ compare_to_baseline_urls['parent'] }}">compare to baseline run from
+      parent commit</a>
+    </p>
+  {% endif %}
+  {% if compare_to_baseline_urls['fork_point'] %}
+    <p class="fs-5">
+      <i class="bi bi-file-diff"></i> <a href="{{ compare_to_baseline_urls['fork_point'] }}">compare to baseline run from
+      fork point commit</a>
+    </p>
+  {% endif %}
+  {% if compare_to_baseline_urls['latest_default'] %}
+    <p class="fs-5">
+      <i class="bi bi-file-diff"></i> <a href="{{ compare_to_baseline_urls['latest_default'] }}">compare to latest
+      baseline run on default branch</a>
```

Comment:
This will complicate this code (and we could defer this to a new issue, it's a bit of scope creep), but it would be fantastic if we had a label or something that marked which comparison we believe most in. Something like a little `default` or something on:

* parent if the commit is on the default branch
* fork point if the commit is on a non-default branch

This way it's clear to folks looking at this which comparison we firmly believe is the best one for the data they are looking at.

Reply:
This is a good idea and shouldn't be too complicated. Right now,

1. if the run isn't associated with a commit, only `latest_default` can be found
    - `parent` and `fork_point` don't apply because it's not on the git graph
2. if the run is on the default branch, only `parent` can be found
    - `fork_point` doesn't make sense and we [decided](https://github.com/conbench/conbench/pull/1078#discussion_r1174160609) not to find `latest_default` because analysis could get confusingly circular
3. if the run is on a PR branch, all three can be found

So we'll only need to "recommend" one in the last case, in which case the recommendation will always be `fork_point`.

Reply:
_nods_ I would also expect that a(n oldish) run on default might also have `latest_default` too? If a commit after the current one has benchmarks. That's totally fine, and I can imagine a case where someone might want to do that, but we should definitely warn that that's not our recommended, and instead the parent commit is the recommended one. right?

Reply:
Yeah, now that we have an official recommendation in the webapp it's probably less dangerous to return the `latest_default` baseline for default-branch commits too. I'll do this in a followup PR since it'll have a bigger scope.